### DISCUSSION TITLE
please,which directory does MACOS version save the blochchain files?

### DISCUSSION THREAD

**Discussion by mrmaozedong** [Chia-Network/chia-blockchain#6261] [Support]
I deleted chia,but can not find the blochchain files.
please.

**Comment by kettlewarrior** [comment]
At a guess you want the keys? If so, looks like:

```
/Users/${USER}/.chia/mainnet/config/ssl/
```

**Reply by mrmaozedong** [reply]
> At a guess you want the keys? If so, looks like:
> 
> ```
> /Users/${USER}/.chia/mainnet/config/ssl/
> ```

thank you,I just want to find if there are remaining files.

**Reply by joelrb** [reply]
blockhain files are here: /Users/${USER}/.chia/mainnet/db/

if you mean executable files it's in /Applications/Chia.app/Contents/Resources/app.asar.unpacked/daemon

**Reply by mrmaozedong** [reply]
> blockhain files are here: /Users/${USER}/.chia/mainnet/db/
> 
> if you mean executable files it's in /Applications/Chia.app/Contents/Resources/app.asar.unpacked/daemon

thx

**Comment by BrandtH22** [comment]
@mrmaozedong , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
